providers/shared/components/apigateway/id.ftl

```diff
@@ -244,6 +244,37 @@ object.
             {
                 "Names" : "Tracing",
                 "Children" : tracingChildConfiguration
+            },
+            {
+                "Names" : "Image",
+                "Description" : "Control the source of the image that is used for the function",
```

Comment:
This description was applicable to `lambda` implementation of these attributes, but could use updated wording for the apigateway